app_legacy/Helpers/render/achievement.php

```diff
@@ -69,16 +69,18 @@ function renderAchievementTitle(?string $title, bool $tags = true): string
     if (!$title) {
         return '';
     }
-
     if (!Str::contains($title, '[m]')) {
         return $title;
     }
-    $span = '';
+
+    $missableTag = '';
     if ($tags) {
-        $span = '<span class=\'tag missable\' title=\'Missable\'><abbr>[<b>m</b>]</abbr></span>';
+        $missableTag = "<span class='tag missable' title='Missable'><abbr>[<span>m</span>]</abbr></span>";
     }
+    $title = str_replace('[m]', '', $title);
+    $title = str_replace('  ', ' ', $title);
 
-    return trim(str_replace('[m]', $span, $title));
+    return "$title $missableTag";
```

Comment:
The space should be part of `$missableTag`. It won't be necessary for most achievements.